contracts/Automate.sol

```diff
@@ -166,6 +167,43 @@ contract Automate is Gelatofied, Proxied, AutomateStorage, IAutomate {
         );
     }
 
+    function exec1BalanceSimple(
+        address _taskCreator,
+        address _execAddress,
+        bytes memory _execData,
+        bytes32 _taskId,
+        Gelato1BalanceParam calldata _oneBalanceParam,
+        bool _revertOnFailure,
+        bool _singleExec
```

Comment:
those booleans are adding 32 bytes of payload each

Reply:
We could potentially pack it into a uint256 and unpack with bitwise manipulation but IMO it gets kinda hacky to save ~120 gas (simulated on tenderly)

Reply:
Yes I'm not fan of the hackiness neither... 
The L1 gas gain is not relevant here, it's more about tx payload bytes reduction for L2s.

Could you maybe run some simulation on an OP stack chain, to see the current gain and try to have a more concrete estimation of those bytes gain impact